app/helpers/api/messages_helper.rb

```diff
@@ -0,0 +1,2 @@
+module Api::MessagesHelper
```

Comment:
- [ ] It would be better if we have a working linter checks to check for errors and warnings. Although you have the files, you are to ensure the linter checks works and shows in the PR